documentation/internal/development/Workflow.md

```diff
@@ -151,9 +151,7 @@ We are using 2 options for local states:
 
 ##### Example of local state
 
-**TODO**: Rename ApplicationPage => ApplicationPageWrapper (if we decide that we want to keep the wrapper thingy).
-
-**To be created** => In ApplicationPageWrapper:
+In ApplicationPageWrapper:
 
 - List of all questions and responses
```

Comment:
It's not in the diff, but I notice at line 249 there is a spelling error in the `setup/Apollo-client.md` link

Reply:
Fixed. Thanks for noticing that.